<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q3MeshBuilder</class>
 <widget class="QWidget" name="Q3MeshBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,3">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="plotLayout">
     <item>
      <widget class="Q3Plot" name="plotWidget" native="true">
       <zorder></zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="userLayout" stretch="0,0">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="addElementLayout">
       <item>
        <widget class="QToolButton" name="pointButton">
         <property name="text">
          <string>точка</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pointConnectionButton">
         <property name="text">
          <string>отрезок</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="addWidgetsLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="snapToGrid">
         <property name="text">
          <string>привязка к сетке</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q3Plot</class>
   <extends>QWidget</extends>
   <header>q3plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
